src/sql_injection/tokenize_query.rs

```diff
@@ -10,10 +10,5 @@ pub fn tokenize_query(sql: &str, dialect: i32) -> Vec<Token> {
     the escaping" ~ https://github.com/sqlparser-rs/sqlparser-rs/blob/main/src/tokenizer.rs#L591-L620
     */
     let mut tokenizer = Tokenizer::new(dialect.as_ref(), sql).with_unescape(false);
-    match tokenizer.tokenize() {
-        Ok(tokens) => tokens, // Return the tokens if successful
-        Err(_e) => {
-            Vec::new() // Return empty vector if unsuccessfull.
```

Comment:
I use this place often to add a `dbg!(_e)` statement 😬

Reply:
Changed back and added a linter ignore comment.